shoov/modules/custom/shoov_restful/plugins/restful/node/ShoovEntityBaseNode.php

```diff
@@ -53,6 +53,16 @@ protected function imageProcess($value) {
       }
       return $output;
     }
+
+    $defaultStyles = shoov_screenshot_image_default_styles();
```

Comment:
I'd prefer to harcode them in https://github.com/shoov/shoov/blob/fc643475ca0731685dcac3fe5a3a29424750b5e1/shoov/modules/custom/shoov_restful/plugins/restful/node/screenshots/1.0/ShoovScreenshotsResource.class.php#L24